### PR TITLE
[NUI] Fix build warning CA1062

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- Need to verify whether the argument is null
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1062

### API Changes ###
- N/A

### CODE REVIEW

File: src/Tizen.NUI/src/internal/ItemContainer.cs [hunk 1]
```diff
@@ -197,7 +197,14 @@ public ItemContainerEnumerator(ItemContainer collection)
...
                 currentObject = null;
-                currentSize = collectionRef.Count;
+                if (collectionRef != null)
+                {
+                    currentSize = collectionRef.Count;
+                }
+                else
+                {
+                    currentSize = 0;
+                }
```
Comment (huiyueun) [inline src/Tizen.NUI/src/internal/ItemContainer.cs:207]: How about using a '??' ?
currentSize = collectionRef?.Count ?? 0;

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 2]
```diff
@@ -2032,9 +2032,12 @@ public LayoutTransition LayoutTransition
                 {
                     _layoutTransitions = new Dictionary<TransitionCondition, TransitionList>();
                 }
-                LayoutTransitionsHelper.AddTransitionForCondition(_layoutTransitions,value.Condition,value, true);
+                if (value != null)
+                {
+                    LayoutTransitionsHelper.AddTransitionForCondition(_layoutTransitions,value.Condition,value, true);
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:2037]: follow C# code foramt. (Ctrl + K,F in VS)

File: src/Tizen.NUI/src/public/GLWindow.cs [hunk 3]
```diff
@@ -120,13 +120,16 @@ public Size2D WindowSize
...
+                if (value != null)
+                {
+                    global::System.IntPtr intPtr = Interop.GLWindow.GlWindow_GetPositionSize(swigCPtr);
+                    Rectangle val = new Rectangle(intPtr, true);
+                    Rectangle ret = new Rectangle(val.X, val.Y, value.Width, value.Height);
+                    val.Dispose();
+
+                    Interop.GLWindow.GlWindow_SetPositionSize(swigCPtr, Rectangle.getCPtr(ret));
+                    ret.Dispose();
+                }
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/GLWindow.cs:132]: If the value is null, do not we need a log message or an exception?

File: src/Tizen.NUI/src/public/KeyValue.cs [hunk 4]
```diff
@@ -77,22 +77,25 @@ public object OriginalKey
             }
             set
             {
-                _originalKey = value;
-                if (value is int || value is Int32)
+                if (value != null)
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/KeyValue.cs:80]: If the value is null, do not we need a log message or an exception?

File: src/Tizen.NUI/src/public/XamlBinding/ContentPage.cs [hunk 5]
```diff
@@ -109,7 +109,14 @@ public ContentPage(Window win)
...
             Root.HeightResizePolicy = ResizePolicyType.FillToParent;
 
-            win.Add(Root);
+            if (win != null)
+            {
+                win.Add(Root);
+            }
+            else
+            {
+                NUIApplication.GetDefaultWindow().Add(Root);
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/XamlBinding/ContentPage.cs:118]: If the parameter window is null, would not it be better not to act than to attach it to the main window?

Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/XamlBinding/ContentPage.cs]: Accepted your opinion! :)

**Review by Seoyeon2Kim — COMMENTED** [review]
I updated the files as Huiyu's comments.
But, I'm wondering that it would be better to change null check conditionals to the similar ways.

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by huiyueun — APPROVED** [review]

**Review by bshsqa — APPROVED** [review]
ImageView, View, Capture checked.

**Pull request merged**